packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -29,7 +40,18 @@ import { Alert } from '@patternfly/react-core';
 </React.Fragment>
 ```
 
-### Variations
+### Alert variations
+
+PatternFly supports several properties and variations that can be used to add extra content to an alert. 
+
+* As demonstrated in the 1st variation below, use the `actionLinks` property to add one or more `AlertActionLink` components that place links beneath the alert message. You must pass in `href` and `component="a"` properties to have an `AlertActionLink` act as a proper link, rather than as a button.
+
+* As demonstrated in the 2nd variation below, use a native HTML `<a>` element to add links within an alert message.
+
+* As demonstrated in the 3rd and 4th variations below, use the `actionClose` property to add an `AlertActionCloseButton` component, which can be used to manage and customize alert dismissals. `actionClose` can be used with or without the presence of `actionLinks`.
+
+* As demonstrated in the 5th and 6th variations below, order headings by their level and avoid skipping heading levels when possible to avoid confusion. For example, a heading of an h2 level should not be followed directly by an h4.
```

Comment:
Could we mention the `titleHeadingLevel` property here, similar to the other bullets, and wrap the h2 and h4 in backticks?